libayatana-appindicator/libayatana-appindicator-gtk3.json

```diff
@@ -0,0 +1,101 @@
+{
+    "name": "libayatana-appindicator",
+    "buildsystem": "cmake-ninja",
+    "config-opts": [
+        "-DENABLE_BINDINGS_MONO=NO",
+        "-DENABLE_BINDINGS_VALA=NO",
+        "-DENABLE_GTKDOC=NO"
+    ],
+    "modules": [
+        "../intltool/intltool-0.51.json",
+        {
+            "name": "libdbusmenu",
+            "buildsystem": "autotools",
+            "build-options": {
+                "cflags": "-Wno-error",
+                "env": {
+                    "HAVE_VALGRIND_FALSE": "#",
+                    "HAVE_VALGRIND_TRUE": ""
```

Comment:
no. this should be a config-opt. But it seems that `--disable-tests` does the trick.

does that even work?

Reply:
This was required because disable-tests means HAVE_VALGRIND never gets set, needs a patch to fix that. But libdbusmenu is dead upstream so not really any chance of getting the patch in.